main.nf

```diff
@@ -52,7 +53,7 @@ workflow.onComplete {
 		.stripIndent()
 
 	base = csv.getBaseName()
-	logFile = file("/fs1/results/cron/logs/" + base + ".complete")
+	logFile = file("${CRONDIR}/outlog.complete")
```

Comment:
Note to self: Restore this (although this would be great to have in the config instead)

Reply:
Yes, it's better if it comes from the config, I have done similar thing in DSL2 SomaticPanelPipeline

Reply:
Yes, at the moment it needs to be updated in each new branch to prevent emails being sent to Viktor. But I think it will be in a different PR, as it would need to be looked over how it is used in production.